book/basic-features/utterances.md

```diff
@@ -4,10 +4,14 @@
 
 [Utterances 🔮](https://utteranc.es/) is a lightweight open-source widget which allows you, your colleages and your students to discuss stuff in a blog post in your book. It is build on GitHub issues, so requires a GitHub repository, although the book can be hosted anywhere (so also on GitLab).
 
-The [utterances website](https://utteranc.es/) clearly explains the required steps. One thing to take care of is how you want to do the issue-mapping with respect to multiple deployed versions (draft version, student version, archived versions):
- 
- - `Issue title contains page pathname`/`Issue title contains page URL` is useful if you want to isolate the comments on each of the branches.
- - `Issue title contains page title` is useful if you want to show the all comments on all different branches.
+The [utterances website](https://utteranc.es/) clearly explains the required steps. Three things to take care of are:
+1. With the current setup of the [deploy book workflow on GitHub](../external/deploy-book-workflow/README.md), this widget only works on the primary branch.
+2. The baseurl needs to be defined in the template:
```

Comment:
in the config.yml file of the template*

Reply:
and then how can they find the baseurl, the url of their repo

Reply:
Ah, that should indeed be just config, I'll change it

Reply:
Is it better now? @xjulie0x ?